x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch_instructions.js

```diff
@@ -0,0 +1,252 @@
+          Set xpack.monitoring.elasticsearch.collection.enabled to false on each node in the production cluster.
+        </p>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+          language="curl"
+        >
+          {`
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.elasticsearch.collection.enabled": false
+  }
+}
+          `}
+        </EuiCodeBlock>
+      </Fragment>
+    )
+  };
+
+  const installMetricbeatStep = {
+    title: 'Install Metricbeat on the same node as Elasticsearch',
+    children: (
+      <Fragment>
+        <p>
+          Follow <EuiLink to="https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html">the instructions here</EuiLink>
+        </p>
+      </Fragment>
+    )
+  };
+
+
+  const enableMetricbeatModuleStep = {
+    title: 'Enable and configure the Elasticsearch module in Metricbeat',
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+            metricbeat modules enable elasticsearch
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <p>Then, configure the module</p>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+- module: elasticsearch
+  metricsets:
+    - ccr
+    - cluster_stats
+    - index
+    - index_recovery
+    - index_summary
+    - ml_job
+    - node_stats
+    - shard
+  period: 10s
+  hosts: ["${esMonitoringUrl}"]
+  xpack.enabled: true
+`}
+        </EuiCodeBlock>
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: 'Configure metricbeat to send to the monitoring cluster',
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+output.elasticsearch:
+  hosts: ["${esMonitoringUrl}"]
+`}
+        </EuiCodeBlock>
+      </Fragment>
+
+    )
+  };
+
+  const startMetricbeatStep = {
+    title: 'Start Metricbeat',
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
+./metricbeat -e
+`}
+        </EuiCodeBlock>
+      </Fragment>
+
+    )
+  };
+
+  let migrationStatusStep = null;
+  if (product.isInternalCollector || product.isPartiallyMigrated) {
+    let status = null;
+    if (hasCheckedMigrationStatus) {
+      const lastInternallyCollectedTimestamp = formatDateTimeLocal(product.lastInternallyCollectedTimestamp);
+
+      if (product.isPartiallyMigrated) {
+        status = (
+          <Fragment>
+            <EuiSpacer size="m"/>
+            <EuiCallOut
+              size="s"
+              color="warning"
+              title={`We still see data coming from the default collection of Elasticsearch.
+              Make sure you disable that before moving forward.`}
+            >
+              <p>Last internal collection occurred at {lastInternallyCollectedTimestamp}</p>
```

Comment:
When these warnings appear, can you change the step to 'warning'? `status: 'warning'`

Reply:
Nice, good thought!